#goal definition
bool store
---
#result definition
int32 images_acquired
---
#feedback
int32 images_acquired
string storage_path